### ISSUE TITLE
"Client initialization failed" on opening file in another workspace folder

### ISSUE THREAD

**Issue by Songworks** [wkillerud/vscode-scss#52]
Hi,

- VS Code Version: 1.77.3
- SCSS IntelliSense Version: v2.11.1
- Operating System: Ubuntu 22.04.2 LTS

Reproducible Case:

basic file structure:
```
project/
project/somesass.code-workspace
project/foldera/testA.scss
project/folderb/testB.scss
```

Files contain nothing except the `somesass.code-workspace`, which contains:

```json
{
    "folders": [
       {
            "name": "root",
            "path": "./"
        },
        {
            "name": "Folder A",
            "path": "./foldera"
        },
        {
            "name": "Folder B",
            "path": "./folderb"
        }
    ]
}

```

If I open this workspace and edit a scss file within one of the folders the extension and what it does works fine.
But once I open a file in any other folder (regardless of filetype) I get the following error message:

```
Client initialization failed. Error: command '_somesass.applyExtractCodeAction' already exists at m.registerCommand (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:98:124270) at Object.registerCommand (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:114:22984) at /home/user/.vscode/extensions/somewhatstationery.some-sass-2.11.1/dist/node-client.js:2:588644 at /home/user/.vscode/extensions/somewhatstationery.some-sass-2.11.1/dist/node-client.js:2:589585
```

I also get the notification `[Folder B] Starting Some Sass server` which seems to be stuck in loading and I have to restart vscode to get it working again.
Note the folder name prefixed to the notification? Only saw that just now while writing.

Not sure if it's a incompatability with another extension . Haven't gotten around to testing any further yet. End of the day now for me.

Cheers!

PS: Thanks for a great extension. Works wonderfully so far (well, except this I guess :o) )! :+1:

**Comment by wkillerud** [comment]
Thanks for the bug report :) I'll look into it when I get the time.

**Comment by wkillerud** [comment]
Should be fixed in [`2.11.2`](https://github.com/wkillerud/vscode-scss/releases/tag/2.11.2). Thanks again for the report, and the reproducible case :)
